cordova-common/package.json

```diff
@@ -28,6 +28,7 @@
     "cordova-registry-mapper": "^1.1.8",
     "elementtree": "^0.1.6",
     "glob": "^5.0.13",
+    "jasmine": "^2.5.2",
```

Comment:
can you move this down to devDependencies instead and also remove jasmine-node from there